uber/templates/fields/attendee.html

```diff
@@ -1040,7 +1040,7 @@ <h4>{{ tier.name }}</h4>
 
 
 {% set emergency_contact %}
-{% if not admin_area or c.HAS_REG_ADMIN_ACCESS %}
+{% if not attendee.is_new or not admin_area or c.HAS_REG_ADMIN_ACCESS %}
```

Comment:
Sanity check: do you want the ``not`` here?  Wouldn't we prefer to show it if they ARE new so that it's possible to enter this field for new attendees?

Reply:
Oh! You're right! Good catch, let me fix that.